argtable3/.build-test-rules.yml

```diff
@@ -0,0 +1,4 @@
+argtable3/test_apps:
+  enable:
+    - if: IDF_TARGET in ["esp32", "esp32c3"]
+      reason: "Sufficient to test on one Xtensa and one RISC-V target"
```

Comment:
I think we should also disable the test for IDF version < 6.0 in case of any conflict with the argtable3 inside the IDF.